src/Trakx.Kaiko.ApiClient/Registration/HttpClientRegistration.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using Microsoft.Extensions.DependencyInjection;
+using Polly;
+using Polly.Contrib.WaitAndRetry;
+using Polly.Extensions.Http;
+using Serilog;
+using Trakx.Utils.Apis;
+
+namespace Trakx.Kaiko.ApiClient;
+
+internal static class HttpClientRegistration
+{
+    internal static IServiceCollection ConfigureHttpClient<TClient,
```

Comment:
I took this out of the T4 template generation and it works well. However, Codacy is asking me to add parameters for TClient and TImplementation. 

<https://app.codacy.com/gh/trakx/kaiko-api-client/pullRequest?prid=10893365>

I can't do that.
If there's a different way to write this generic code, please help. If not, can we disable the Codacy alerts for this file?

Reply:
ignored